apps/frontend/src/context/MockContext.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+declare global {
+  interface Window {
+    setMock: (b?: boolean) => void;
+    mockHr: number | null;
+    mockWatt: number | null;
+  }
```

Comment:
I tried to have functions for the mockHr and mockWatt-setting also, but I ran into trouble regarding updating and stuff. So i just went with fields